pkg/transport/transport.go

```diff
@@ -18,11 +18,13 @@ package transport
 
 import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 type Transport interface {
 	InformerSynced() bool
 	WrapObjects([]*unstructured.Unstructured) *unstructured.Unstructured // wrap multiple objects into a single wrapped object.
+	GetWrappedObjectGVR() schema.GroupVersionResource
```

Comment:
Perhaps add a TODO on what we discussed (the work that results in this interface eventually containing only wrapObjects)

Reply:
I hope to complete this part today or the day after. otherwise I'll add a TODO